compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmLower.kt

```diff
@@ -107,6 +107,7 @@ val jvmPhases = namedIrFilePhase(
             jvmTypeOperatorLoweringPhase then
             flattenStringConcatenationPhase then
             foldConstantLoweringPhase then
+            computeStringTrimPhase then
```

Comment:
@JakeWharton 
As order of lowers is sufficient here, please add also some tests with constant inlining and append operations, e.g.:
```
private const val SPACES = "             "


    """
        ${SPACES + SPACES}
        Hello,
        World
        ${SPACES + SPACES}
    """.trimIndent()

(SPACES + """
        ${SPACES + SPACES}
        Hello,
        World
        ${SPACES + SPACES}
    """ + SPACES + ).trimIndent()

```

Reply:
Updated!